rbx_types/src/basic_types.rs

```diff
@@ -142,6 +142,9 @@ impl Matrix3 {
             z: Vector3::new(0.0, 0.0, 1.0),
         }
     }
+    pub fn new(x: Vector3, y: Vector3, z: Vector3) -> Self {
+        Self { x, y, z }
+    }
```

Comment:
Can we move `new` above `identity` and throw a blank line between them? This is sort of the default constructor, which I like to have first conventionally.